### ISSUE TITLE
Doesn't work with yarn workspaces

### ISSUE THREAD

**Issue by LukasBombach** [nklayman/vue-cli-plugin-electron-builder#19]
Hi I just ran

```
npm install -g @vue/cli
vue create app # and I selected my own settings in the interactive CLI here
cd app
vue add electron-builder
yarn serve:electron
```

and I got this error:

```
yarn run v1.3.2
$ vue-cli-service serve:electron
{ Error: spawn /Users/lbombach/Projekte/emc-vue/packages/app/node_modules/.bin/webpack-dev-server ENOENT
    at _errnoException (util.js:1003:13)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:201:19)
    at onErrorNT (internal/child_process.js:389:16)
    at process._tickCallback (internal/process/next_tick.js:152:19)
From previous event:
    at /Users/lbombach/Projekte/emc-vue/node_modules/graceful-fs/polyfills.js:287:18
    at FSReqWrap.oncomplete (fs.js:167:5)
From previous event:
    at startRenderer (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/out/dev/WebpackDevServerManager.js:149:17)
    at /Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/src/dev/dev-runner.ts:43:7
    at Generator.next (<anonymous>)
    at runCallback (timers.js:756:18)
    at tryOnImmediate (timers.js:717:5)
    at processImmediate [as _immediateCallback] (timers.js:697:5)
From previous event:
    at DevRunner.start (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/src/dev/dev-runner.ts:32:14)
    at /Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/src/dev/dev-runner.ts:142:19
    at Generator.next (<anonymous>)
From previous event:
    at main (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/out/dev/dev-runner.js:132:18)
    at Object.<anonymous> (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/src/dev/dev-runner.ts:145:1)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
    at Function.Module._load (module.js:505:3)
    at Module.require (module.js:604:17)
    at require (internal/module.js:11:18)
    at runInDevMode (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/out/cli.js:46:3)
    at Object._yargs.default.command.command.command.command.command.argv [as handler] (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/out/cli.js:22:440)
    at Object.runCommand (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/node_modules/yargs/lib/command.js:235:44)
    at Object.parseArgs [as _parseArgs] (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/node_modules/yargs/yargs.js:1046:30)
    at Object.get [as argv] (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/node_modules/yargs/yargs.js:989:21)
    at Object.<anonymous> (/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/out/cli.js:22:455)
    at Module._compile (module.js:660:30)
    at Object.Module._extensions..js (module.js:671:10)
    at Module.load (module.js:573:32)
    at tryModuleLoad (module.js:513:12)
  errno: 'ENOENT',
  code: 'ENOENT',
  syscall: 'spawn /Users/lbombach/Projekte/emc-vue/packages/app/node_modules/.bin/webpack-dev-server',
  path: '/Users/lbombach/Projekte/emc-vue/packages/app/node_modules/.bin/webpack-dev-server',
  spawnargs:
   [ '--color',
     '--env.autoClean=false',
     '--config',
     '/Users/lbombach/Projekte/emc-vue/node_modules/electron-webpack/webpack.renderer.config.js' ] }
Renderer WDS exited with code -2
```

I also tried adding electron

```
yarn add electron
```

but no luck. Any ideas?

**Comment by LukasBombach** [comment]
I found the issue: It fails with yarn workspaces. I have a monorepo and this lib tries to spawn `node_modules/.bin/webpack-dev-server` but because I use yarn workspaces `webpack-dev-server` is located in `../node_modules/.bin/webpack-dev-server`

**Comment by nklayman** [comment]
I think this a a problem with electron-webpack as it is what uses webpack-dev-sever. In v1.0.0 I will be dropping electron-webpack so it should work then. There should be a beta within two weeks.

**Comment by LukasBombach** [comment]
Coolz thanks. Im gonna close this issue then.

**Comment by nklayman** [comment]
I just released the first v1.0.0 alpha! Go ahead and try it again with the new version and let me know if it works. Thanks.

**Comment by LukasBombach** [comment]
Aweseome! Thank you!
